bash/gast_to_code/bash_gast_to_code_converter.py

```diff
@@ -16,5 +16,16 @@ def handle_log_statement(gast):
         return "echo"
     
     def handle_func_call(gast):
+        # bash has no way to directly print arrays. This logic returns an error for such behavior
+        # TODO: maybe we should run code after translation to make sure it compiles and throw errors if it does NOT
```

Comment:
I'd rather have this TODO in a taiga ticket. This seems applicable to all converters.

Reply:
I like the idea, but may not work  with unsuported code blocks in the output.

Reply:
Added taiga ticket- removing TODO